pkg/runctl/run_manager.go

```diff
@@ -145,7 +145,7 @@ func (c *runManager) prepareRunNamespace(ctx *runContext) error {
 	// If something goes wrong while creating objects inside the namespaces, we delete everything.
 	cleanupOnError := func() {
 		if err != nil {
-			c.cleanup(ctx)
+			_ = c.cleanup(ctx)
```

Comment:
Those changes will not remove the checkmarx findings.
We do not handle the error here (and other locations) since we don't want to return the error, we don't want to react on it, and it is already logged in the called function.

Question: Do we want to mention this more explicitly in the code? Or simply switch back to the state before, ignoring the error, and audit the finding in checkmarx?

Reply:
```suggestion
			c.cleanup(ctx) //ignoring error
```